### ISSUE TITLE
Linter can't handle experimental pragmas

### ISSUE THREAD

**Issue by joeykrug** [duaraghav8/Solium#117]
`pragma experimental ABIEncoderV2;` is what it's erroring over.

Error:
```
[FATAL ERROR] An error occurred while linting over 
/home/ubuntu/workspace/source/contracts/Augur.sol: An error occured while parsing the source 
code: Expected "solidity", comment, end of line, or whitespace but "e" found. Line: 2, Column: 8
```

**Comment by duaraghav8** [comment]
@joeykrug Is experimental pragma statement documented somewhere? I couldn't find this feature in Solidity docs, although I did find this [PR](https://github.com/ethereum/solidity/pull/2690) describing `experimental`

**Comment by joeykrug** [comment]
It’s mentioned on the solidity releases page on github

**Comment by elenadimitrova** [comment]
I'm getting this error as well, having just switched on the experimental features.
Although not documented in the solc docs, `experimental` was introduced here https://github.com/ethereum/solidity/issues/2346
And these are the current experimental features supported:
https://github.com/ethereum/solidity/blob/1c85ba10e13879f680579031234c911099433596/libsolidity/ast/ExperimentalFeatures.h

**Comment by duaraghav8** [comment]
1 thing I haven't been able to verify yet is whether you're allowed to specify multiple feature names in the same exp. pragma statement or not. If yes (in [this PR](https://github.com/ethereum/solidity/pull/2690) it says it does), what's the syntax?
Comma-separated & space-separated don't seem to work (solcjs `v0.4.17`)

**Comment by elenadimitrova** [comment]
That PR indeed says it does but cannot get some suggestions I thought of to compile and all samples are using separate experimental declarations.

**Comment by duaraghav8** [comment]
so axic replied. Only single feature name allowed per statement.

**Comment by duaraghav8** [comment]
Support has been added, please run dependancy update for Solium (in case you find it tricky, just re-install the linter).

**Comment by elenadimitrova** [comment]
Tested on clean CI build and this still persists.

**Comment by duaraghav8** [comment]
This issue has been resolved in `v1.0.2`.
If you're still using `v0` (run `solium -V` to check), I highly recommend to move to v1.
Its backward compatible & comes with automatic code formatting, more flexible configuration format & security lint rules.
See [docs](http://solium.readthedocs.io/en/latest/)

**Comment by elenadimitrova** [comment]
I no longer get the error however there is an issue with the `import` rule as if you have experimental pragmas at the top of the file followed by imports, it throws:
```
^-- ERROR: Import statement must be at the top of the file.
```

If I move the imports at the top of the file followed by the pragmas, it then throws the expected
```
^-- WARNING: Pragma Directive "pragma solidity ^0.4.17;" should only be at the top of the file.
```

The only combination that works is if the `pragma solidity ^0.4.17;` is at the top, followed by the imports followed by the experimental pragmas. Which is clearly incorrect.
